helm/content-store/templates/deployment.yaml

```diff
@@ -58,12 +58,24 @@ spec:
           livenessProbe:
             httpGet:
               path: /healthcheck/live
-              port: {{ .Values.containerPort }}
+              port: {{ .Values.appPort }}
             initialDelaySeconds: 30
             periodSeconds: 5
           readinessProbe:
             httpGet:
               path: /healthcheck/ready
-              port: {{ .Values.containerPort }}
+              port: {{ .Values.appPort }}
             initialDelaySeconds: 30
             periodSeconds: 5
+        - name: {{ .Release.Name }}-nginx
```

Comment:
We should include basic liveness and readiness probes for the Nginx container - I'd go with a TCP check rather than HTTP, as per [the Nginx helm chart](https://github.com/bitnami/charts/blob/f88d9fae4eaeba918df2aec7a76efb268e3d0318/bitnami/nginx/templates/deployment.yaml#L165), as any HTTP response means that Nginx is alive and ready.

Reply:
i will look into this and then run some test after implementation.